youtube_dl/extractor/vvvvid.py

```diff
@@ -156,3 +165,56 @@ def f(m):
             'view_count': int_or_none(video_data.get('views')),
             'like_count': int_or_none(video_data.get('video_likes')),
         }
+
+
+class VVVVIDShowIE(VVVVIDIE):
+    _VALID_URL = r'(?P<base_url>%s(?P<show_id>\d+)/(?P<id>[^/]+))/?(?:$|[\?&].*$)?$' % VVVVIDIE._VALID_URL_BASE
+    _TESTS = [{
+        'url': 'https://www.vvvvid.it/show/156/psyco-pass',
+        'info_dict': {
+            'id': '156',
+            'title': 'Psycho-Pass',
+            'description': 'md5:94d572c0bd85894b193b8aebc9a3a806',
+        },
+        'playlist_count': 46,
+    }]
+
+    def _real_extract(self, url):
+        base_url, show_id, video_id = re.match(self._VALID_URL, url).groups()
+
+        response = self._download_info(
+            '%s/seasons/' % show_id, video_id)
+
+        show_infos = self._download_info(
+            '%s/info/' % show_id, video_id)
+
+        show_title = show_description = None
+        if show_infos:
+            show_title = show_infos['data'].get('title')
+            show_description = show_infos['data'].get('description')
+
+        entries = []
+        for season in response['data']:
+            if season['episodes']:
+                for episode in season['episodes']:
```

Comment:
reduce the indentation.
```suggestion
            episodes = season.get('episodes') or []
            for episode in episodes:
```